x-pack/plugins/ml/public/file_datavisualizer/components/edit_flyout/options/option_lists.js

```diff
@@ -12,7 +12,10 @@ export const FORMAT_OPTIONS = [
   // 'xml',
 ];
 
+export const CUSTOM_DROPDOWN_OPTION = 'custom';
+
 export const TIMESTAMP_OPTIONS = [
+  'custom',
```

Comment:
Could you use the `CUSTOM_DROPDOWN_OPTION` constant here?

Reply:
Yep 👍 Thanks! 
Updated in 51c237651d7a6baebf826a47672823461f8055d9